### DISCUSSION TITLE
How to customize accent color?

### DISCUSSION THREAD

**Discussion by ashtrath** [bonzibudd/Fluent-Metro#172] [Q&A]
Is it possible to use custom accent color, searchbar and item hover? i love this skins because i can use custom start menu background color, it would be nice if i can change the color with things i mentioned above ^^

EDIT: also is there anyway to decrease the start menu size and completely remove the sub-menu?

**Comment by bonzibudd** [comment]
Hello, on version 1.5.3 I added some highlight color options, which lets you customize the color of some elements:
 - Submenu background
 - New programs highlight
 - Search box border
 - Search header highlight
 
This doesn’t affect item selections, but I might consider that option in the future.

To decrease the size of the menu, check the [Fluent-Metro wiki](https://github.com/bonzibudd/Fluent-Metro/wiki#change-menu-size). There’s not a way to get rid of the submenu, but you can customize the right-hand menu items so they open directly instead of expanding to the sub-menu.
